Gemfile.base

```diff
@@ -91,10 +91,9 @@ gem 'analytics-ruby', require: false
 group :development, :test do
   gem 'bootsnap', '~> 1.3'
 
-  platform :mri_23, :mri_24, :mri_25 do
-    gem 'pry-byebug', '>= 3.7.0', require: false
-    gem 'pry-stack_explorer', require: false
-  end
+  gem 'pry-byebug', '>= 3.7.0', require: false
```

Comment:
You can keep ruby 2.4 and 2.5 because it does not work on JRuby

Reply:
Okay, I will add it back